src/nodejs/analytics/analytics.cpp

```diff
@@ -456,8 +494,23 @@ void TNodeJsSVC::fit(const v8::FunctionCallbackInfo<v8::Value>& Args) {
                 JsModel->Model = TSvm::TLinModel(SvmModel->GetWgtV(), SvmModel->GetThresh());
             }
             else if (JsModel->Algorithm == "LIBSVM") {
-                JsModel->Model = TSvm::LibSvmSolveClassify(VecV, ClsV, JsModel->SvmCost, JsModel->SvmUnbalance,
-                    TQm::TEnv::Debug, TQm::TEnv::Error);
+              int type;
+              if (JsModel->SvmType == "default") type = C_SVC;
```

Comment:
Can we replace libsvm defined consts with our enums (what I mentioned above) for things like `C_SVC`? There should be no libsvm stuff outside svm.cpp/h